### DISCUSSION TITLE
Possibility to change the default names, off fields.

### DISCUSSION THREAD

**Discussion by ihr-it-projekt** [directus/directus#23337] [Feature Requests]
### Summary

When you create a new collection, you can add default field like user_created. It will be great if you can change that name with a new default value so you don't have to change it any time when you create a new collection.

### Motivation

Enables us not to change it any time when we create a new collection.

### Detailed Design

Add an option to edit default names of new creative collection items.

### Must Have:
- default names can be changed
- new default names are used when creating a new collection

### Drawbacks

---

### Alternatives

---

### Adoption Strategy

---

**Comment by Stretsh** [comment]
I guess https://github.com/directus/directus/discussions/18824 (with more upvotes) would already solve this.
